integration-tests/framework.yaml

```diff
@@ -11,6 +11,7 @@ logging:
 chainlink_image:
 chainlink_version:
 chainlink_env_values:
+  LOG_LEVEL: 'debug'
```

Comment:
This change is just to ensure env vals don't eval to nil, required for future work.